tevm/docs/state/functions/deepCopy.md

```
@@ -8,7 +8,11 @@
 
 > **deepCopy**(`baseState`): () => `Promise`\<[`BaseState`](../type-aliases/BaseState.md)\>
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)